.github/workflows/tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: python setup.py develop
 
       - name: Run tests
-        run: coverage run -m pytest -W error
```

Comment:
I disagree with this change. 
- IMO, the test is supposed to work without warnings. It shall therefore fail upon warnings.
- if you feel that a particular warning is acceptable, pleas adapt the command line arguments to ignore that particular warning or error.

Reply:
Decided that tests with warnings should fail in principle.
Also decided to leave this change (and ignore warnings) for now, but open an issue to restore `-W error` later.